main/load-tx-rlp.zkasm

```diff
@@ -95,7 +95,7 @@ shortGasPrice:
                                         :CALL(addBatchHashData)
 
 endGasPrice:
-        A                               :MSTORE(txGasPrice)
+        A                               :MSTORE(txGasPriceRLP)
```

Comment:
I would add here the code to set `txGasPriceRLP` into `txGasPrice` that it is done in `process-tx.zkasm`
``` 
$ => A                          :MLOAD(txGasPriceRLP)
        A                               :MSTORE(txGasPrice)
```

Why not use just only `txGasPrice` as a global var?
If it is needed, then update comments on the `vars.zkasm` file